spec/shipcloud/address_spec.rb

```diff
@@ -35,23 +35,27 @@
 
   describe '.create' do
     it 'makes a new POST request using the correct API endpoint' do
-      Shipcloud.should_receive(:request).with(:post, 'addresses', valid_attributes).and_return('data' => {})
+      expect(Shipcloud).to receive(:request).
+        with(:post, 'addresses', valid_attributes, api_key: nil).and_return('data' => {})
+
       Shipcloud::Address.create(valid_attributes)
     end
   end
 
   describe '.find' do
     it 'makes a new GET request using the correct API endpoint to receive a specific address' do
-      Shipcloud.should_receive(:request).with(
-        :get, 'addresses/123', {}).and_return('id' => '123')
+      expect(Shipcloud).to receive(:request).with(
+        :get, 'addresses/123', {}, api_key: nil).and_return('id' => '123')
+
       Shipcloud::Address.find('123')
     end
   end
 
   describe '.update' do
     it 'makes a new PUT request using the correct API endpoint' do
-      Shipcloud.should_receive(:request).with(
-        :put, 'addresses/123', {:street => 'Mittelweg' }).and_return('data' => {})
+      expect(Shipcloud).to receive(:request).with(
+        :put, 'addresses/123', {:street => 'Mittelweg' }, api_key: nil).and_return('data' => {})
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.